src/Tribe/PUE/Checker.php

```diff
@@ -673,14 +677,14 @@ public function request_update() {
 		 * @param $plugin_data
 		 */
 		public function in_plugin_update_message( $plugin_data ) {
-			$plugininfo = $this->json_error;
+			$plugin_info = $this->json_error;
```

Comment:
If `$this->json_error` is now deprecated, should we use its replacement `$this->plugin_info` instead? I realize both are _currently_ the same, but in general we shouldn't use our own deprecated fields or functions.